mcs/tools/xbuild/Makefile

```diff
@@ -106,12 +106,12 @@ NUGET_BUILDTASKS_REPO_DIR=$(topdir)/../external/nuget-buildtasks
 
 install-nuget-targets:
 	$(MKINSTALLDIRS) $(DESTDIR)$(NUGET_BUILDTASKS_TARGETS_DIR)
-	$(MKINSTALLDIRS) $(DESTDIR)$(MSBUILD_EXTENSIONS_15_DIR)/Imports/Microsoft.Common.props/ImportBefore
-	$(MKINSTALLDIRS) $(DESTDIR)$(MSBUILD_EXTENSIONS_15_DIR)/Microsoft.Common.targets/ImportAfter
+	$(MKINSTALLDIRS) $(DESTDIR)$(MSBUILD_EXTENSIONS_CURRENT_DIR)/Imports/Microsoft.Common.props/ImportBefore
```

Comment:
While it doesn't make any difference _at the moment_ - this looks broken from a long-term maintainability perspective.

You shouldn't ever _install_ anything into `Current/`, but instead use whichever the current version prefix is.

Just imagine the situation a couple of years down the line when `Current` points to something that's more recent than our current `15.0`, but we are still maintaining this particular code in some long-term stable branch.